src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.wasm.cs

```diff
@@ -119,6 +98,19 @@ public static void SetBorder(UIElement element, Thickness thickness, Brush brush
 						break;
 				}
 			}
+
+			if (cornerRadius == CornerRadius.None)
+			{
+				element.ResetStyle("border-radius", "overflow");
+			}
+			else
+			{
+				var outer = cornerRadius.GetRadii(element.RenderSize, thickness).Outer;
+				var borderRadius = $"{outer.TopLeft.X}px {outer.TopRight.X}px {outer.BottomRight.X}px {outer.BottomLeft.X}px / {outer.TopLeft.Y}px {outer.TopRight.Y}px {outer.BottomRight.Y}px {outer.BottomLeft.Y}px";
```

Comment:
Let's use an explicit javascript invocation instead of formatting on the C# side (which may also have failed because of non-invariant formatting), let's do the same with the reset above.

Reply:
I made the change for corner radius specifically.
Not sure how much else we want to move?